### DISCUSSION TITLE
useTooltipTrigger and default value delay prop

### DISCUSSION THREAD

**Discussion by Agile-fox** [adobe/react-spectrum#4194] [Q&A] [answered]
Hey,
I've been working with `useTooltipTrigger`  and I made it work but I found 2 things related to the `delay`prop which I couldn't solve.

1. Is there any way to set up a custom delay for disappearing of the tooltip?
2. Can we set up inside a component a default value for `delay` prop instead of standard 1500 which come from hooks for case when user don't pass `delay` prop? I tried to set this up as I was doing with aria label but in this case it don't give any effect. 

```
const { triggerProps, tooltipProps } = useTooltipTrigger({...props, delay: 0}, state, triggerRef)
```

You can check behavior here:
https://codesandbox.io/s/vigilant-buck-5fwuhw?file=/src/App.js

**Comment by reidbarber** [comment, marked as answer]
I modified your example to do both of the things you mentioned: https://codesandbox.io/s/ecstatic-brown-hj8giz?file=/src/App.js

You can just set your default value in props before it gets passed to the state hook or aria hook.

**Reply by Agile-fox** [reply]
Thanks, this is exactly for what I was looking for :) I edited title a little bit to better resemble the question.
